build.assets/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile makes the "build box": the container used to build official
 # releases of Teleport and its documentation.
-FROM ubuntu:20.04
+FROM ubuntu:18.04
```

Comment:
Add a comment at the top explaining why we're using 18.04 rather than 20.04 now, just so we don't forget this in 6 months when we come to bump the buildbox version again :)

Reply:
This has caught myself off guard for old Debian boxes. I know it adds more complexity, but I wonder if we should split old Distros to use 18.0x and keep using newer versions for master / latest versions.

Reply:
I think this downgrade is acceptable as we're not using any other components of the 18.04 buildbox - just the built `teleport`, `tctl` and `tsh` binaries that are generated.

If this change makes it so the glibc version is low enough for CentOS 7 and other older distros then that's good - we should keep our required glibc version as low as possible so that we can work with the most distros.

Reply:
Added